From 9dbf25cfe0c3f47c021f7c2c6494f8c682945c84 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 14 Oct 2018 19:25:00 +0300
Subject: [PATCH] Revert "Linux: I2C bus update number"

This reverts commit 827297f1cc8b34d928580ccaa017860e24cfb9d6.
cannot use both masters since bus is combined
---

diff --git a/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc b/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc
index 9d47028..3641d87 100755
--- a/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc
+++ b/board/habanalabs/common/rootfs_overlay/etc/init.d/S99misc
@@ -6,7 +6,7 @@
 case "$1" in
   start)
 	echo "Starting misc..."
-	echo powr1220 0x25 > /sys/bus/i2c/devices/i2c-1/new_device
+	echo powr1220 0x25 > /sys/bus/i2c/devices/i2c-0/new_device
 	#echo ir35221 0x40 > /sys/bus/i2c/devices/i2c-0/new_device
 	echo 0 > /sys/module/rcupdate/parameters/rcu_cpu_stall_suppress
 	for tz in /sys/class/thermal/thermal_zone*;
